src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -147,6 +170,13 @@ func (manager Manager) Delete(id string) error {
 
 // Restores Rancher Desktop to the state saved in a snapshot.
 func (manager Manager) Restore(id string) error {
+	// Before doing anything, ensure that the snapshot is complete
+	completeFilePath := filepath.Join(manager.Paths.Snapshots, id, completeFileName)
+	if _, err := os.Stat(completeFilePath); errors.Is(err, os.ErrNotExist) {
```

Comment:
Same as above: if the `os.Stat()` returns **any** error, do we really want to try to restore from that snapshot?